src/components/AccountSecurity/LoginPage.tsx

```diff
@@ -365,7 +365,7 @@ class LoginPage extends Component<Props, State> {
                 </div>
                 <div className="row">
                   <div className="col-10 pl-0">
-                    <Link to="/find-organization">
+                    <Link to="/find-organizations">
```

Comment:
I fixed this bug already -- please merge in the latest master

Reply:
ok